TestFoundation/TestBundle.swift

```diff
@@ -234,41 +243,41 @@ class TestBundle : XCTestCase {
             ("test_bundleLoad", test_bundleLoad),
             ("test_bundleLoadWithError", test_bundleLoadWithError),
             ("test_bundleWithInvalidPath", test_bundleWithInvalidPath),
-            ("test_bundlePreflight", test_bundlePreflight),
```

Comment:
Why was this test removed?

Reply:
I changed the name of the test function to `test_bundleFindExecutables`. For the `testBundle()` the test will try a `Bundle.preflight()` which must success and for the `withEachPlaygroundLayout` bundles the test will try to locate the executable (trying to `preflight()` them will always fail because the executable are empty file).

Reply:
The `test_bundleFindExecutables` does not appear to be added here, though.

Reply:
Yes, it does.

Reply:
That makes more sense, but I'm not quite sure why we're not testing that invalid executables can't be preflighted anymore.